modules/installation-proxy-remove.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * installing/insall_config/enable-cluster-wide-proxy.adoc
+
+[id="installation-proxy-remove_{context}"]
+= Removing proxy from the cluster
+
+The `cluster` proxy object cannot be deleted. To remove proxy from a cluster,
+remove all `spec` fields from the proxy object.
+
```

Comment:
Since `spec` can not be removed, I suggest providing an example like so:
```
$ oc get proxy/cluster -o yaml
apiVersion: config.openshift.io/v1
kind: Proxy
metadata:
  name: cluster
spec: {}
status: {}
```